test/utils.js

```diff
@@ -25,6 +25,8 @@ const FIREFOX_PREFERENCES = {
   // toolbox.
   "devtools.chrome.enabled": true,
   "devtools.debugger.remote-enabled": true,
+  "devtools.debugger.prompt-connection": false,
+  "general.warnOnAboutConfig": false
```

Comment:
Great that you added this in. It was on my improvements list. :) I also submitted a [PR](https://github.com/mozilla/shield-studies-addon-utils/pull/92) to set these two prefs in the SSAU repo.

Reply:
Good!